.github/workflows/integration-tests.yml

```diff
@@ -27,6 +27,13 @@ jobs:
           # Install python dependencies
           pip install --user openshift pygithub
 
+          # Install apt packages
+          apt-get -y update
+          apt-get -y install jq curl
+
+          # Add certificates to trusted list
+          cat  operator-pipeline-images/certs/* >> /etc/ssl/certs/ca-certificates.crt
```

Comment:
Is there a reason you don't use the standard way to install ca certs by using update-ca-certificates?

Reply:
Only that it is not installed in default Ubuntu image, but I can install it with apt if you think it's worth it.

Reply:
Yes, I think it's worth installing the tool.